eng/template-engine.yml

```diff
@@ -50,9 +50,9 @@ jobs:
         pool:
           vmImage: 'macOS-latest'
         ${{ if eq(variables['System.TeamProject'], 'public') }}:
-          helixTargetQueue: OSX.1015.Amd64.Open
+          helixTargetQueue: OSX.1100.Amd64.Open
```

Comment:
Not related to the PR: we don't use helix - this should be removed. @GangWang01 could you please check it? Thank you.